tests/controls/infobar.cpp

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        tests/controls/infobar.cpp
+// Purpose:     wxInfoBar  tests
+// Author:      Blake Madden
+// Created:     2025-6-02
+// Copyright:   (c) 2025 Blake Madden
+///////////////////////////////////////////////////////////////////////////////
+
+// ----------------------------------------------------------------------------
+// headers
+// ----------------------------------------------------------------------------
+
+#include "testprec.h"
+
+
+#ifndef WX_PRECOMP
+    #include "wx/app.h"
+#endif // WX_PRECOMP
+
+#include "wx/infobar.h"
+
+#include <memory>
+
+namespace
+{
```

Comment:
This seems inconsistent with the other tests, so I'll remove it if you don't mind.

Reply:
Yeah, I saw other files doing that so I wasn't sure. Totally OK with removing that.